dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -226,6 +226,16 @@
 			status = "disabled";
 			label = "UART_8";
 		};
+
+		rtc: rtc@58004000 {
+			compatible = "st,stm32-rtc";
+			reg = <0x58004000 0x300>;
```

Comment:
The bank for RTC/BKP  is 0x400 wide   0x58004000 - 0x580043FF 
as specified in the Reference manual STM32H742, STM32H743/753 and STM32H750

Reply:
Thanks @FRASTM , fixed